collect.py

```diff
@@ -203,6 +203,18 @@ def screenshot(driver, file_path):
     image.save(file_path)
 
 
+def get_code(driver, file_path):
+    wait_loaded(driver)
```

Comment:
Since both this function and screenshot are using **wait_loaded**, let's move it in **get_screenshot_and_domtree**. We don't need to wait for the page to be loaded twice.